lib/instances.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const db = {
+    register,
+    getDb,
+    getModel,
+    getModels
+};
+
+let instances = [];
+
+Object.defineProperty(db, 'dbs', {
+    configurable: false,
+    enumerable: true,
+    get: () => instances.reduce((dbs, instance) => {
+        dbs[instance.name] = instance.db;
+        return dbs;
+    }, {})
+});
+
+module.exports = db;
+
+function register(name, db) {
+    const instance = findDb(name);
+    if (!instance) {
+        instances.push({
+            name,
+            db
+        });
+    } else {
+        instance.db = db;
+    }
+}
+
+function getModel(dbName, modelName) {
+    let instance;
+    let model;
+    
+    if (modelName) {
+        instance = getDb(dbName);
+    } else {
+        modelName = dbName;
+        instance = getDb();
+    }
+    
+    model = instance.models[modelName];
+    
+    if (!model) {
+        throw new Error('Database model not found');
+    }
+    
+    return model;
+}
+
+function getModels(dbName) {
+    return getDb(dbName).models;
+}
+
+function findDb(dbName) {
+    return instances.find(db => db.name === dbName);
+}
+
+function getDb(dbName) {
+    let instance;
+    if (dbName) {
+        instance = findDb(dbName);
+    } else {
+        instance = instances[0];
+    }
+    
+    if (!instance) {
+        throw new Error('Database not found');
```

Comment:
Please use the same message as for the server implementation `hapi-sequelizejs cannot find the ${name} database instance`

Reply:
Fixed in 898c7885954ff24f67da4c35eb157e040fe37d3d